ext/ext.md

```diff
@@ -0,0 +1,56 @@
+# _ext - Index of Distribution Extensions
+
+## Summary
+
+This base extension is used to discover and return an array of extensions.
+
+## Reference Explanation
+
+The base extension may be queried through a `GET` to discover available
+extensions.
+
+Registry level extensions may be discovered with a standard GET as follows.
+
+```HTTP
+    GET /v2/_ext/
+```
+
+Repository level extensions may be discovered with a standard GET as follows.
+
+```HTTP
+    GET /v2/{name}/_ext/
+```
+
+The base extension returns an array of supported extensions with details of the
+endpoints as shown below.
+
+```HTTP
+200 OK
+Content-Length: <length>
+Content-Type: application/json
+
+{
+    "extensions: [
+        "_<ns>/<ext>/<component>",
```

Comment:
Sorry, missed a slash in my comment, fixed. This is for the response to the API request. If we return an array of strings, it can't be easily extended later if we want to include other metadata about extensions.